customer/src/main/java/no/unit/nva/customer/get/FetchChannelClaimHandler.java

```diff
@@ -0,0 +1,63 @@
+package no.unit.nva.customer.get;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static no.unit.nva.customer.Constants.defaultCustomerService;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.util.UUID;
+import no.unit.nva.customer.get.response.ChannelClaimResponse;
+import no.unit.nva.customer.service.CustomerService;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import nva.commons.apigateway.exceptions.UnauthorizedException;
+import nva.commons.core.JacocoGenerated;
+
+public class FetchChannelClaimHandler extends ApiGatewayHandler<Void, ChannelClaimResponse> {
+
+    private static final String CLAIM_NOT_FOUND = "Channel claim with identifier %s not found";
+    private static final String NOT_VALID_IDENTIFIER = "Provided channel claim identifier is not valid!";
+    private static final String IDENTIFIER_PATH = "identifier";
+    private final CustomerService customerService;
+
+    @JacocoGenerated
+    public FetchChannelClaimHandler() {
+        this(defaultCustomerService());
+    }
+
+    public FetchChannelClaimHandler(CustomerService customerService) {
+        super(Void.class);
+        this.customerService = customerService;
+    }
+
+    @Override
+    protected void validateRequest(Void unused, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        userIsAuthorized(requestInfo);
+    }
+
+    private static void userIsAuthorized(RequestInfo requestInfo) throws UnauthorizedException {
+        requestInfo.getCurrentCustomer();
```

Comment:
For now checking simply if user is authorized. Should take in use new access right once commons will be bumped in all projects.